modules/tensor_mechanics/tests/mohr_coulomb/tests

```diff
@@ -126,6 +133,13 @@
     rel_err = 1.0E-5
     abs_zero = 1.0E-3
   [../]
+  [./uni_axial1_small_strain]
+    type = 'CSVDiff'
+    input = 'uni_axial1_small_strain.i'
+    csvdiff = 'uni_axial1.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-3
```

Comment:
Thanks Andy! Then I would say check it in as is.
On Thu, May 7, 2015 at 2:31 AM Andy Wilkins <notifications@github.com>
wrote:

> In modules/tensor_mechanics/tests/mohr_coulomb/tests
> <https://github.com/idaholab/moose/pull/5030#discussion_r29827616>:
>
> > @@ -126,6 +133,13 @@
> >      rel_err = 1.0E-5
> >      abs_zero = 1.0E-3
> >    [../]
> > +  [./uni_axial1_small_strain]
> > +    type = 'CSVDiff'
> > +    input = 'uni_axial1_small_strain.i'
> > +    csvdiff = 'uni_axial1.csv'
> > +    rel_err = 1.0E-5
> > +    abs_zero = 1.0E-3
>
> Thanks @friedmud <https://github.com/friedmud> for your essay. I fluffed
> around with the BCs etc in this particular problem and didn't come up with
> anything good at all. The original is the most succinct and converges the
> fastest to the correct solution.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5030/files#r29827616>.
>